codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoder.java

```diff
@@ -110,6 +119,8 @@ protected String decodeName0(ByteBuf in) {
         return decodeName(in);
     }
 
+    protected String decodeCharString0(ByteBuf in) { return decodeCharString(in); }
```

Comment:
make this multiple lines. Also please add javadocs and I also wonder why this needs to be protected.